README.md

```diff
@@ -96,7 +96,7 @@ Please check [Tips for using Azure CLI effectively](https://docs.microsoft.com/e
 - [Generic resource commands - `az resource`](https://docs.microsoft.com/en-us/cli/azure/use-cli-effectively#generic-resource-commands---az-resource)
 - [REST API command - `az rest`](https://docs.microsoft.com/en-us/cli/azure/use-cli-effectively#rest-api-command---az-rest)
 - [Quoting issues](https://docs.microsoft.com/en-us/cli/azure/use-cli-effectively#quoting-issues)
-- [Work behind a proxy](https://docs.microsoft.com/en-us/cli/azure/use-cli-effectively#work-behind-a-proxy)
+- [Proxy blocks connection](https://learn.microsoft.com/en-us/cli/azure/use-azure-cli-successfully-troubleshooting#error-sslerror-bad-handshakecertificate-verify-failed-proxy-blocks-connection)
```

Comment:
@dbradish-microsoft, I think this header is too long and results in a very long URL https://learn.microsoft.com/en-us/cli/azure/use-azure-cli-successfully-troubleshooting#error-sslerror-bad-handshakecertificate-verify-failed-proxy-blocks-connection:

![image](https://github.com/Azure/azure-cli/assets/4003950/fdacf558-e4b1-4a5c-911f-aab8b4649121)

Also, `bad handshake...certificate verify failed` may not be the only error that can be caused by proxy blocking connection.

Reply:
@jiasli, I don't disagree. I'm trying to find the best optimum search phrase for SEO regardless of the URL. Might you have a magic cube to give me a list of other error messages for "proxy blocks connection"?

Reply:
@jiasli, does [this PR](https://github.com/MicrosoftDocs/azure-docs-cli/pull/4451) solve the problem?  The long H2 gives us SEO if a customer searches for that error message, and the proxy H2 is the original H2 syntax and is subject-oriented vs focusing on a single error.